src/frontend/src/components/Searching.tsx

```diff
@@ -1,20 +1,28 @@
 import { TextField, ToggleButton, ToggleButtonGroup } from '@mui/material';
 import axios from 'axios';
-import React, { useState, useRef, useCallback, KeyboardEvent } from 'react';
+import React, {
+   useState,
+   useRef,
+   useCallback,
+   KeyboardEvent,
+   useEffect,
+} from 'react';
 import SingleCard from './SingleCard';
 import useSearchApartment from './useSearchApartment';
 
 export default function Searching() {
    const [query, setQuery] = useState('');
    const [pageNum, setPageNum] = useState(1);
+   const [press, setPress] = useState(false);
+   const emptyarray: string[] = [];
+   const [selected, setSelected] = useState(emptyarray);
    const { loading, error, apartments, hasMore } = useSearchApartment(
```

Comment:
Why is this called in `Searching.tsx`?

Reply:
It receives the apartments from the GET requests, and whether they have more, is loading, etc.

Reply:
> It receives the apartments from the GET requests, and whether they have more, is loading, etc.

Search should only display apartment names though, the actual apartments are populated differently.

Reply:
![Screen Shot 2022-10-26 at 12 31 28 PM](https://user-images.githubusercontent.com/59807670/198096116-f70a96fa-0e0f-49d0-b098-4053ffe87022.png)
Do are you saying I should have been doing the gray circle not the red circle?

Reply:
> The search strings are displayed similar to what happens in google search or other social media searches: through a pop up.
>
<img width="1024" alt="Screen Shot 2022-10-26 at 12 33 34 PM" src="https://user-images.githubusercontent.com/59807670/198096407-f17bb7b1-ae22-4c4f-b0da-89cab5078f53.png">

Is this what you mean by pop up?

Reply:
The `search` thing in mainpage corresponds to the grey circle, similar to what I explained below. This corresponds to the `search_apartments()` method in `Mainpage.py`.

The list of apartments are displayed by default on the page, which the user can sort based on price and rating. This corresponds to the various `apartments_*.py` methods in `Mainpage.py`.

So yes, search should be the grey circle, the red circle is different.